llvm/utils/release/build_llvm_release.bat

```diff
@@ -192,8 +196,6 @@ set "stage0_bin_dir=%build_dir%/build32_stage0/bin"
 set cmake_flags=^
   %common_cmake_flags% ^
   -DLLVM_ENABLE_RPMALLOC=OFF ^
-  -DLLDB_TEST_COMPILER=%stage0_bin_dir%/clang.exe ^

```

Comment:
Not needed? I supposed we've never been able to run the tests successfully anyway..